packages/faustwp-cli/index.ts

```diff
@@ -111,8 +111,11 @@ import {
    * Spawn a child process using the args captured in argv and continue the
    * standard i/o for the Next.js CLI.
    */
+  const nextjsCommand = process.platform === 'win32' ? 'next.cmd' : 'next';
```

Comment:
Does `win32` cover 64 bit arch as well or do we need to add `win64`?

Reply:
@blakewilson Yes, 64 bit will use `win32` as `win64` is not a valid option.

https://nodejs.org/api/process.html#processplatform